### DISCUSSION TITLE
Touch for GUI elements.

### DISCUSSION THREAD

**Discussion by cbradburne** [MyreMylar/pygame_gui#203] [Ideas]
I've just started making my app 'touch friendly', namely on a 22" touch screen connected to a Raspberry Pi.

I've used pygame's FINGERDOWN and so on for some elements, but any pygame-gui element, like buttons or drop down menus don't yet support touch as far as I'm aware. 

Are there any plans to incorporate touch?

It's not a priority, just a hope it's on a timeline :)

**Comment by MyreMylar** [comment]
Yeah right now the entire GUI system is primarily mouse driven as that's what I was using in my apps. Touch fits alongside console controllers and 'keyboard only' as input options I want to increase support for. 

It's definitely somewhere on the roadmap before 1.0.
